bdk_core/src/descriptor_tracker.rs

```diff
@@ -12,81 +12,81 @@ use miniscript::{
     descriptor::DerivedDescriptorKey, psbt::PsbtInputExt, Descriptor, DescriptorPublicKey,
 };
 
+/// A [DescriptorTracker] is a collection of maps and lists, that contains
```

Comment:
Don't link to the thing you are doc commenting. Always use backticks around type names (even if they are in a link).